src/components/animalBreedType.jsx

```diff
@@ -14,6 +14,7 @@ const AnimalBreedType = ({
   animalSubCategories,
   shouldShow,
 }) => {
+  const animalBreedType= rest.formik.values.animalBreedType;
```

Comment:
there is an easier way to do this if your variable name is the same as the last property

```js
const { animalBreedType } = rest.formik.values;
```

See

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Object_destructuring

Reply:
done